### ISSUE TITLE
Dashboard Data is Blank

### ISSUE THREAD

**Issue by Vortec4800** [parse-community/parse-dashboard#628]
Make sure these boxes are checked before submitting your issue -- thanks for reporting issues back to Parse Dashboard!

- [X] You're running version >=1.0.21 of Parse Dashboard.

- [X] You're running version >=2.2.24 of Parse Server.

- [X] You've searched through [existing issues](https://github.com/ParsePlatform/parse-dashboard/issues?utf8=%E2%9C%93&q=). Chances are that your issue has been reported or resolved before.

#### Environment Setup

The Dashboard is running on a separate machine from the Mongo database and from the Parse server. All servers are running Ubuntu Server 14.04.5 on Digital Ocean.

#### Steps to reproduce
Run server then run dashboard. Open Browser on Dashboard and select any Class.

#### Logs/Trace
There are no visible error messages. Not sure how to get further logs from Dashboard - only how to show logs from Parse server.

Basically the dashboard is empty and shows no data. The scheme does load, and sometimes on a fresh load I can get a single screen-full of data. But no count values populate on the dashboard, and if I change to another class I don't get any more data. I was running Dashboard version x.19, just updated to x.22 in case that fixed my issue, but it's no different.

I'm not sure how to get more diagnostic info from the Dashboard that might tell me what the issue is. Parse.com dashboard (this was a migration project) does still show my data, and the app itself does work like normal, so it seems like this issue is isolated to the Dashboard.

**Comment by Vortec4800** [comment]
Another thing to mention, the database is very large, I've got a couple collections with around 10M records.

**Comment by TouchTitans** [comment]
I'm running into the same issue. Heroku shows the data in MongoDB, Parse Dashboard shows it is connected but doesn't show any collections.

**Comment by blerchy** [comment]
I am having this exact same issue, except my database is very small, with < 20 entries (early in development).

Another issue is that none of the menu buttons seem to do anything, so for example if I try to delete a class I can click the "Edit" menu button which gives a dropdown, but clicking "Delete this class" from the dropdown does _nothing_, doesn't even dismiss the dropdown. :confused:

**Comment by alexrmacleod** [comment]
Same issue, running parse-dash on my laptop, displaying data from my ubuntu 14.04 droplet. it was working fine 5 mins ago. then suddenly no more data. All classes won't display, I wonder what caused it.

**Comment by flovilmart** [comment]
> it was working fine 5 mins ago

Not sure what do you with that, can you guys give more info about the state of your database, parse-server logs when running with VERBOSE=1, network traces from the Chrome inspector etc ...

**Comment by rodtaborda** [comment]
I had this problem also. I saved a JPEG as a PFFile via an IOS app and after that the Class stopped (I couldn't see data and make any changes). I removed the same row using an Android App. Now it's working again.

**Comment by skynext** [comment]
Hi, I've the exact same issue on a relatively small database. I run my parse server on Windows Azure and I use parse dashboard 1.0.28

**Comment by flovilmart** [comment]
@Vortec4800, you mention a very large collection, did you index on the sorting keys correctly?

**Comment by hatimmakki** [comment]
I have the same issue, happened after I saved a new row with byte[] in File column.

**Comment by RajeshDhalange** [comment]
No classes! No data! It was working fine when I closed my work yesterday, now when I resume work and check in dashboard there is no data and classes as well. It's really terrible.

**Comment by flovilmart** [comment]
Check the inspector on your browser, the server logs on your server, and open an issue describing your problem.

**Comment by RajeshDhalange** [comment]
Thank for reply, more strange is server console and parse dashboard log section don't have any server error or log everything looks like a new setup, now the question is what scenario I describe for an open issue.  I am trying to open mongo db but not getting success can you guide me to open mongodb on CentOS

**Comment by hatimmakki** [comment]
I got the same issue, and this what i got in chrome console:

```
dashboard.bundle.js:9 iterable.length has been deprecated, use iterable.size or iterable.count(). This warning will become a silent error in a future version. Error
    at Ue.get (https://parse-dashboard.back4app.com/bundles/dashboard.bundle.js:9:30362)
    at t.<anonymous> (https://parse-dashboard.back4app.com/bundles/dashboard.bundle.js:44:9674)
    at s (https://parse-dashboard.back4app.com/bundles/dashboard.bundle.js:52:29411)
    at Generator._invoke (https://parse-dashboard.back4app.com/bundles/dashboard.bundle.js:52:30805)
    at Generator.e.(anonymous function) [as next] (https://parse-dashboard.back4app.com/bundles/dashboard.bundle.js:52:29590)
    at s (https://parse-dashboard.back4app.com/bundles/dashboard.bundle.js:52:29411)
    at t (https://parse-dashboard.back4app.com/bundles/dashboard.bundle.js:52:29671)
    at https://parse-dashboard.back4app.com/bundles/dashboard.bundle.js:52:29782
    at <anonymous>
get @ dashboard.bundle.js:9
(anonymous) @ dashboard.bundle.js:44
s @ dashboard.bundle.js:52
(anonymous) @ dashboard.bundle.js:52
e.(anonymous function) @ dashboard.bundle.js:52
s @ dashboard.bundle.js:52
t @ dashboard.bundle.js:52
(anonymous) @ dashboard.bundle.js:52
Promise.then (async)
t @ dashboard.bundle.js:52
(anonymous) @ dashboard.bundle.js:52
t @ dashboard.bundle.js:8
r @ dashboard.bundle.js:52
n @ dashboard.bundle.js:52
e.(anonymous function) @ dashboard.bundle.js:52
N.async @ dashboard.bundle.js:53
value @ dashboard.bundle.js:44
(anonymous) @ dashboard.bundle.js:44
s @ dashboard.bundle.js:52
(anonymous) @ dashboard.bundle.js:52
e.(anonymous function) @ dashboard.bundle.js:52
s @ dashboard.bundle.js:52
t @ dashboard.bundle.js:52
(anonymous) @ dashboard.bundle.js:52
Promise.then (async)
t @ dashboard.bundle.js:52
(anonymous) @ dashboard.bundle.js:52
t @ dashboard.bundle.js:8
r @ dashboard.bundle.js:52
n @ dashboard.bundle.js:52
e.(anonymous function) @ dashboard.bundle.js:52
N.async @ dashboard.bundle.js:53
value @ dashboard.bundle.js:44
value @ dashboard.bundle.js:44
updateComponent @ dashboard.bundle.js:34
receiveComponent @ dashboard.bundle.js:34
receiveComponent @ dashboard.bundle.js:11
_updateRenderedComponent @ dashboard.bundle.js:34
_performComponentUpdate @ dashboard.bundle.js:34
updateComponent @ dashboard.bundle.js:34
receiveComponent @ dashboard.bundle.js:34
receiveComponent @ dashboard.bundle.js:11
updateChildren @ dashboard.bundle.js:33
_reconcilerUpdateChildren @ dashboard.bundle.js:35
_updateChildren @ dashboard.bundle.js:35
updateChildren @ dashboard.bundle.js:35
_updateDOMChildren @ dashboard.bundle.js:34
updateComponent @ dashboard.bundle.js:34
receiveComponent @ dashboard.bundle.js:34
receiveComponent @ dashboard.bundle.js:11
_updateRenderedComponent @ dashboard.bundle.js:34
_performComponentUpdate @ dashboard.bundle.js:34
updateComponent @ dashboard.bundle.js:34
receiveComponent @ dashboard.bundle.js:34
receiveComponent @ dashboard.bundle.js:11
updateChildren @ dashboard.bundle.js:33
_reconcilerUpdateChildren @ dashboard.bundle.js:35
_updateChildren @ dashboard.bundle.js:35
updateChildren @ dashboard.bundle.js:35
_updateDOMChildren @ dashboard.bundle.js:34
updateComponent @ dashboard.bundle.js:34
receiveComponent @ dashboard.bundle.js:34
receiveComponent @ dashboard.bundle.js:11
_updateRenderedComponent @ dashboard.bundle.js:34
_performComponentUpdate @ dashboard.bundle.js:34
updateComponent @ dashboard.bundle.js:34
receiveComponent @ dashboard.bundle.js:34
receiveComponent @ dashboard.bundle.js:11
_updateRenderedComponent @ dashboard.bundle.js:34
_performComponentUpdate @ dashboard.bundle.js:34
updateComponent @ dashboard.bundle.js:34
receiveComponent @ dashboard.bundle.js:34
receiveComponent @ dashboard.bundle.js:11
_updateRenderedComponent @ dashboard.bundle.js:34
_performComponentUpdate @ dashboard.bundle.js:34
updateComponent @ dashboard.bundle.js:34
performUpdateIfNecessary @ dashboard.bundle.js:34
performUpdateIfNecessary @ dashboard.bundle.js:11
l @ dashboard.bundle.js:8
perform @ dashboard.bundle.js:12
perform @ dashboard.bundle.js:12
perform @ dashboard.bundle.js:8
S @ dashboard.bundle.js:8
closeAll @ dashboard.bundle.js:12
perform @ dashboard.bundle.js:12
batchedUpdates @ dashboard.bundle.js:34
o @ dashboard.bundle.js:8
dispatchEvent @ dashboard.bundle.js:34
dashboard.bundle.js:39 Uncaught (in promise) TypeError: n.url is not a function
    at m (dashboard.bundle.js:39)
    at d._constructComponentWithoutOwner (dashboard.bundle.js:34)
    at d._constructComponent (dashboard.bundle.js:34)
    at d.mountComponent (dashboard.bundle.js:34)
    at Object.mountComponent (dashboard.bundle.js:11)
    at _.mountChildren (dashboard.bundle.js:35)
    at _createInitialChildren (dashboard.bundle.js:34)
    at _.mountComponent (dashboard.bundle.js:34)
    at Object.mountComponent (dashboard.bundle.js:11)
    at Object.updateChildren (dashboard.bundle.js:33)
```
